libs/ccf-shared/src/lib/services/data-source/api-endpoint.service.ts

```diff
@@ -101,11 +104,39 @@ function spatialSceneNodeReviver(nodes: RawSpatialSceneNode[]): SpatialSceneNode
   providedIn: 'root',
 })
 export class ApiEndpointDataSourceService implements DataSource {
+  private readonly http = inject(HttpClient);
+
   constructor(
     private readonly api: DefaultService,
     private readonly globalConfig: GlobalConfigState<ApiEndpointDataSourceOptions>,
   ) {
     globalConfig.getOption('hubmapToken').subscribe(buster$);
+    combineLatest([
+      globalConfig.getOption('dataSources'),
+      globalConfig.getOption('filter'),
```

Comment:
Oh also, filter will change a lot as they use the tool. We need to capture the 'initial filter'. I'm not sure if we have logic in here to catch that.